README.md

```diff
@@ -21,6 +22,26 @@ can be used to resolve [phyloreferences](http://phyloref.org):
   `input.owl` by comparing their resolution with the expected resolution recorded
   in the file.
 
+# Build and execution instructions
+
+JPhyloRef can be built from source by running `mvn package` from the source directory.
```

Comment:
By _source directory_ do you mean the `src/` directory, some directory where one placed (downloaded?) the source code, or the root directory of this repository?

Reply:
I mean the root directory of this repository -- I started by writing "root directory", but then I was concerned that that might be confusing. Do you think that would be better? Maybe "root directory of this repository" to avoid ambiguity?

Reply:
> Maybe "root directory of this repository" to avoid ambiguity?

Yes, that's what I normally write.

Reply:
Done! Fixed in 121719a55c90.